app/components/view-section-heading/_section-heading.scss

```diff
@@ -2,6 +2,6 @@
   background-color: $color_nhsuk-grey-5;
   border-bottom: 2px solid $color_nhsuk-blue;
   margin-top: nhsuk-spacing(9);
-  margin-bottom: nhsuk-spacing(3);
+  margin-bottom: nhsuk-spacing(5) !important;
```

Comment:
what does this do?

Reply:
over rides a class that nhs put on somewhere so there is a space below the blue line

Reply:
probably don't need the important as it only gets put on the last child
but when I was looking at the title component in isolation it was the last child 😆

Reply:
removed it